pkg/controllers/disruption/controller.go

```diff
@@ -61,23 +56,11 @@ const pollingPeriod = 10 * time.Second
 
 var errCandidateDeleting = fmt.Errorf("candidate is deleting")
 
-// waitRetryOptions are the retry options used when waiting on a NodeClaim to become ready or to be deleted
-// readiness can take some time as the node needs to come up, have any daemonset extended resoruce plugins register, etc.
-// deletion can take some time in the case of restrictive PDBs that throttle the rate at which the node is drained
-func waitRetryOptions(ctx context.Context) []retry.Option {
-	return []retry.Option{
-		retry.Context(ctx),
-		retry.Delay(2 * time.Second),
-		retry.LastErrorOnly(true),
-		retry.Attempts(60),
-		retry.MaxDelay(10 * time.Second), // 22 + (60-5)*10 =~ 9.5 minutes in total
-	}
-}
-
 func NewController(clk clock.Clock, kubeClient client.Client, provisioner *provisioning.Provisioner,
-	cp cloudprovider.CloudProvider, recorder events.Recorder, cluster *state.Cluster) *Controller {
+	cp cloudprovider.CloudProvider, recorder events.Recorder, cluster *state.Cluster, queue *orchestration.Queue) *Controller {
 
 	return &Controller{
+		Queue:         queue,
```

Comment:
```suggestion
		queue:         queue,
```

Any reason that this needs to be publicly exposed?

Reply:
Don't think so. Good call